src/libsyncengine/jobs/network/abstractnetworkjob.cpp

```diff
@@ -319,7 +319,7 @@ bool AbstractNetworkJob::sendRequest(Poco::Net::HTTPSClientSession &session, con
         stream.push_back(session.sendRequest(req));
         if (ioOrLogicalErrorOccurred(stream[0].get())) {
             int err = session.socket().getError();
-            LOG_WARN(_logger, "sendRequest failed for job " << jobId() << " - err=" << err);
+            LOG_WARN(_logger, "sendRequest failed for job " << jobId() << " - socket err=" << err);
```

Comment:
I have been reading the logs attached to redmine issues today and I realize that `Poco::getMessage(int errorCode)` could be useful here.

Reply:
Agreed, its done